app/views/candidate_interface/restructured_work_history/job/_form.html.erb

```diff
@@ -7,11 +7,13 @@
 <% end %>
 
 <div class="app-work-experience__start-date" data-qa="start-date">
-  <%= f.govuk_date_field :start_date, omit_day: true, legend: { text: t('application_form.restructured_work_history.start_date.label'), size: 'm' }, hint: { text: t('application_form.restructured_work_history.start_date.hint_text') } %>
-  <div class="govuk-form-group">
-    <%= f.hidden_field :start_date_unknown, value: false %>
-    <%= f.govuk_check_box :start_date_unknown, true, multiple: false, label: { text: t('application_form.restructured_work_history.start_date_unknown_checkbox') } %>
-  </div>
+  <%= f.govuk_fieldset legend: { text: t('application_form.restructured_work_history.start_date.label'), size: 'm' } do %>
+    <%= f.govuk_date_field :start_date, omit_day: true, legend: nil, hint: { text: t('application_form.restructured_work_history.start_date.hint_text') } %>
```

Comment:
Unfortunately, I don't think this quite works as the `govuk_date_field` still generates a second `<fieldset>` nested within the first one, with the hint now only associated with the second fieldset instead of the outer one.

Possibly this is ok, but I think the recommendation from DAC was to keep the existing single fieldset with its legend and its hint, but to move the checkbox within that fieldset.

I'm not sure that that is currently possible with the way our current `govuk_date_field` helper is set up though, so we'd either have to amend that to allow injection of additional HTML within the fieldset after the date inputs, or else switch to using the `govuk_text_field` helper for both the month and year...

Reply:
<img width="1127" alt="Screenshot 2022-06-08 at 10 08 53" src="https://user-images.githubusercontent.com/30665/172581204-54279960-689c-47e4-a44c-37623d860ab8.png">

Reply:
From what I could tell the recommendation of the DAC was about shared labels?
<img width="662" alt="Screenshot 2022-06-08 at 11 54 01" src="https://user-images.githubusercontent.com/58793682/172599516-4dcb9c31-088f-4564-a6d2-42666ef3a95c.png">. With the change, it now means the screenreader recognises the 'this is the estimate' checkbox as part of the group with the date fields (used voiceover).

Reply:
We've agreed to deploy this change for now, and then do some more research and investigation to see whether it causes any further issues.

Reply:
Also agreed to go back to DAC for further clarity on recommendation, this is a temporary/perm fix depending on response.